ui/main/src/app/business/services/date-time-formatter.service.ts

```
@@ -10,20 +10,23 @@
     private static dateTimeFormat;
     private static dateFnsLocaleOption: {locale: any};
+    private static readonly destroy$ = new Subject<void>();
 
     public static init() {
         DateTimeFormatterService.timeFormat = ConfigService.getConfigValue('settings.timeFormat', 'p');
         DateTimeFormatterService.defaultDateFormat = ConfigService.getConfigValue('settings.dateFormat', 'P');
         DateTimeFormatterService.dateTimeFormat = ConfigService.getConfigValue('settings.dateTimeFormat');
-        ConfigService.getConfigValueAsObservable('settings.locale', 'en').subscribe((locale) =>
-            DateTimeFormatterService.setDateFnsLocaleOption(locale)
-        );
+        this.destroy$.next(); // unsubscribe from previous subscription , only useful for unit tests as they call init more than one time
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)